### PR TITLE
Update docs copied from beats.

### PR DESCRIPTION
**Pull request by simitt**
Add script that
* copies all docs from a local beats folder that are currently listed in the `docs/copied-from-beats/`  folder
* (removed: temporary fixes inconsistencies between beats and apm-server documentation (e.g. removing the monitoring section).)
*  run `make docs` afterwards.

You can trigger the script by running `make update-beats-docs`. 

Please verify the generated documentation afterwards. In case new files are linked, the `make docs` command should fail and you need to manually resolve issues. 

(update info)

### CODE REVIEW

File: docs/copied-from-beats/outputconfig.asciidoc [hunk 1]
```diff
@@ -254,11 +258,13 @@ Example elasticsearch output with `pipelines`:
 
 ["source","yaml"]
 ------------------------------------------------------------------------------
-filebeat.prospectors:
-- paths: ["/var/log/app/normal/*.log"]
+filebeat.inputs:
```
Comment (jalvz) [inline docs/copied-from-beats/outputconfig.asciidoc]: is this relevant to us?

File: docs/copied-from-beats/shared-path-config.asciidoc [hunk 2]
```diff
@@ -14,10 +14,12 @@
...
 The `path` section of the +{beatname_lc}.yml+ config file contains configuration
-options that define where the Beat looks for its files. For example, all Beats
-look for the Elasticsearch template file in the configuration path, Filebeat and
-Winlogbeat look for their registry files in the data path, and all Beats write
-their log files in the logs path.
+options that define where {beatname_lc} looks for its files. For example, {beatname_uc}
+looks for the Elasticsearch template file in the configuration path and writes
+log files in the logs path.
+ifeval::["{beatname_lc}"=="filebeat" or "{beatname_lc}"=="winlogbeat"]
+Filebeat and Winlogbeat look for their registry files in the data path.
```
Comment (jalvz) [inline docs/copied-from-beats/shared-path-config.asciidoc]: irrelevant?

File: docs/copied-from-beats/template-config.asciidoc [hunk 3]
```diff
@@ -82,3 +82,7 @@ setup.template.overwrite: false
 setup.template.settings:
   _source.enabled: false
 ----------------------------------------------------------------------
+
+*`setup.template.append_fields`*:: A list of of fields to be added to the template and Kibana index pattern. experimental[]
+
+NOTE: With append_fields only new fields can be added an no existing one overwritten or changed. This is especially useful if data is collected through the http/json metricset where the data structure is not known in advance. Changing the config of append_fields means the template has to be overwritten and only applies to new indices. If there are 2 Beats with different append_fields configs the last one writing the template will win. Any changes will also have an affect on the Kibana Index pattern.
```
Comment (jalvz) [inline docs/copied-from-beats/template-config.asciidoc]: relevant?

**Comment by roncohen** [comment]
This is great @simitt. Good work 👍

File: docs/copied-from-beats/outputconfig.asciidoc (hunk 1 above)
Comment (simitt) [reply docs/copied-from-beats/outputconfig.asciidoc]: no, and I checked - it doesn't show up in the rendered docs

File: docs/copied-from-beats/template-config.asciidoc (hunk 3 above)
Comment (simitt) [reply docs/copied-from-beats/template-config.asciidoc]: nope you are right, this doesn't make sense.

File: docs/copied-from-beats/shared-path-config.asciidoc (hunk 2 above)
Comment (simitt) [reply docs/copied-from-beats/shared-path-config.asciidoc]: doesn't show up, see `ifeval::["{beatname_lc}"=="filebeat" or "{beatname_lc}"=="winlogbeat"]`

**Comment by graphaelli** [comment]
closes #666
